addon/components/search-help-modal/component.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+import layout from './template';
+
+export default Ember.Component.extend({
+    layout,
+    isOpen: false,
+    currentPath: Ember.computed(function() {
+        return Ember.getOwner(this).lookup('controller:application').currentPath;
```

Comment:
Does this imply that this component requires additional application configuration to use? (special properties on the application controller)

In general we try to keep components reusable by explicitly passing them the info they need, rather than relying on the code of a controller on a different route entirely. I'd recommend refactoring here if possible.